.github/workflows/integration-tests.yml

```diff
@@ -140,5 +145,9 @@ jobs:
 
       # Channels to notify upon workflow failure only
       # notify-slack-channels-upon-workflow-failure: "#channel-here"
+
+      # Report which environment the workflow ran in
+      # chart-release is the term used by dsp-devops tooling to refer to an environment/application pair.
+      relates-to-chart-releases: ${{ steps.test-env.outputs.test-env }}/javatemplate
```

Comment:
Is the value of `relates-to-chart-releases` restricted, or can it be any value? I guess the assumption is that an integration test run must correspond to a chart that was deployed?

Reply:
It is restricted it needs to be a valid `chart-release` selector in sherlock's data model. 
There are a few forms it can take but the most common one and likely only one folks will ever use is `environment_name/service_name` - where service name needs to be the name of the helm chart, not the title of the service's github repo.

Reply:
If someone was trying to set this up such that they were trying to run integration tests against an environment / combination that didn't already exist then they would already be in a larger discussion with devops around that.

Reply:
In this repo, integration tests can run against any environment which is configured using a server config. This config may correspond to a deployed configuration, but that is not  currently enforced by anything in the repo (as far as I'm aware). In other words, the integration test framework doesn't really know about charts right now. They are only conventionally the same.

For example, this repo configures a `local` integration test environment which is what's used to run the tests on PRs. This doesn't correspond to any release. Although since that's run directly from the PR GitHub action there wouldn't be a need to pass its environment to Sherlock. 

See https://github.com/DataBiosphere/terra-java-project-template/tree/main/integration/src/main/resources/servers